inc/class.client.php

```diff
@@ -63,6 +66,16 @@ public function __construct() {
 		return true;
 	}
 
+	function enqueue_displayformat_scripts() {
+
+		wp_register_script('taxopress-frontend.js', STAGS_URL . '/assets/frontend/js/frontend.js', array('jquery'), STAGS_VERSION);
```

Comment:
Remove the dot (.) from the name. Only use either hyphen or underscore to separate name just like we do for others